### ISSUE TITLE
CLI broken: aacli and status

### ISSUE THREAD

**Issue by vwbusguy** [Unicon/shibboleth-idp-dockerized#65]
In the 3.4.3 image, attempting to run `bin/aacli.sh`, `bin/status.sh`, or `version.sh` returns an error: `Error: Could not find or load main class net.shibboleth.idp.cli.CLI`, even with just invoking help.  I'm running these with Shibboleth running - I can fetch the metadata page remotely and the `/status` endpoint is live and appears normal.

We use `aacli.sh` as part of our integration testing.  I additionally, verified that ResolverTest is enabled in the conf/admin/general-admin.xml file.  The same config works in the 3.3.2 image.

**Comment by jtgasper3** [comment]
@vwbusguy ,

You are correct. The 3.4.x shell script's classpath was updated to point libraries in a new location that the installer copies libs into. I haven't had a chance to move the libs into the same spot via the Dockerfile.

Definitely a todo. For now, you can mod the runclass.sh (guessing at the name) to remove `dist` (also guessing at the value) and saving the file. The utilities scripts should then work.

John

**Comment by vwbusguy** [comment]
The missing dist might be related to https://github.com/Unicon/shib-cas-authn3/issues/39 as well.

**Comment by vwbusguy** [comment]
Removing dist still complains about the missing class.  I assume I'll also need to update the classpath as well.

**Comment by jtgasper3** [comment]
Removing the directory doesn’t help, the lib directory needs to get moved 
into it (or similar) OR change the classpath to point to the old location.

**Comment by vwbusguy** [comment]
It looks like setting this in runclass.sh allowed it to work: `LOCALCLASSPATH="$LOCATION/../webapp/WEB-INF/lib/*":$LOCALCLASSPATH`

**Comment by vwbusguy** [comment]
I was able to override it for now by overwriting the runclass.sh during docker build.  Our automated tests against aacli.sh are working again.  Many thanks.  For now, we'll compare for changes with each tag change.  Please let me know if you would like any help testing a more permanent fix.
